addons/website_mail/controllers/main.py

```diff
@@ -37,8 +37,22 @@ def website_message_subscribe(self, id=0, object=None, message_is_follower="on",
             record.sudo().message_subscribe(partner_ids)
             return True
 
-    @http.route(['/website_mail/is_follower'], type='json', auth="public", website=True)
-    def is_follower(self, model, res_id, **post):
+    @http.route(['/website_mail/is_followers'], type='json', auth="public", website=True)
+    def is_followers(self, records, **post):
+        """ Given a list of `models` containing a list of res_ids, return
+            the res_ids for which the user is follower and some practical info.
+
+            param records: {
+                'res.model': [1, 2, 3..],
+                'res.model2': [1, 2, 3..],
+                ..
+            }
+
+            return: [
+                {'is_user': True/False, 'email': 'admin@yourcompany.example.com'},
+                {'res.model': [1, 2], 'res.model2': [1]}
+            ]
+        """
```

Comment:
https://github.com/odoo/odoo/pull/40408#discussion_r366449944 :smile:

Reply:
Meeh. I was not thinking about docstring when writing this method comment but it turn out it looks like a docstring indeed so I'll adapt ;-)